docs/decex/staking.md

```diff
@@ -78,7 +86,7 @@ As soon as the end of the epoch transition is finalized, the next epoch begins.
 
 ## Power Balance 
 
-Farmers who have earned storage rewards nominate operators to execute transactions. This nomination system balances the power between farmers and operators, and both parties share the rewards and the potential penalties (slashing). Farmer-nominated operators get a higher chance to produce blocks proportional to the amount of stake backing them, thus, higher revenues. Farmers have the power to nominate operators they trust to execute transactions properly. On the other hand, operators compete to be nominated by providing good service, maintaining a good reputation within the community, and having reasonable commission. 
+Farmers who have earned storage rewards nominate operators to execute transactions. This nomination system balances the power between farmers and operators, and both parties share the fees and the potential penalties (slashing). Farmer-nominated operators get a higher chance to produce blocks proportional to the amount of stake backing them, thus, higher revenues. Farmers have the power to nominate operators they trust to execute transactions properly. On the other hand, operators compete to be nominated by providing good service, maintaining a good reputation within the community, and having reasonable commission. 
```

Comment:
> balances the power between farmers and operators, and both parties share the fees and the potential penalties (slashing)

This reads as though farmers can be slashed. I assume you mean something like "token holders who nominate"?

> Farmer-nominated operators

And maybe we should switch from implying only farmers can nominate here? We should make it clear it's a use-case for acquiring/holding the token?

I get you're explaining the balance of power but, in isolation, this passage reads as though non-farmer token-holders do not exist.

Reply:
I re-read the notes above about who can be a nominator. Maybe my concerns here are unfounded - your shout :)

Reply:
I'll try to revise this section. Maybe repeat the sentence that anyone can nominate.